build_tools/github_actions/configure_ci.py

```diff
@@ -138,6 +138,8 @@ def contains(cls, val):
     "runtime/src/iree/hal/drivers/cuda/*",
     # Common code likely enough to affect code paths unique to NVIDIA GPUs:
     "compiler/src/iree/compiler/GlobalOptimization/*",
+    # Tests.
+    "tests/e2e/*",
 ]
 
 AMDGPU_PATHS = [
```

Comment:
Might want to choose when to run https://github.com/iree-org/iree/blob/main/.github/workflows/pkgci_regression_test.yml based on similar paths. Or just run all GPU tests all the time... but we'd need runner capacity for that.